Tasks/QuickPerfTest/CltTasksUtility.ps1

```diff
@@ -166,7 +166,7 @@ function ComposeAccountUrl($connectedServiceUrl, $headers)
 	return $elsUrl
 }
 
-function ValidateInputs($websiteUrl, $tfsCollectionUrl, $connectedServiceName)
+function ValidateInputs($websiteUrl, $tfsCollectionUrl, $connectedServiceName, $testName)
```

Comment:
This file is duplicated.. We might need to fix the other two tasks.. long term need to remove this duplication..